src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/ObjectExplorerService.cs

```diff
@@ -406,7 +405,18 @@ internal ExpandResponse QueueExpandNodeRequest(ObjectExplorerSession session, st
         {
             NodeInfo[] nodes = null;
             TreeNode node = session.Root.FindNodeByPath(nodePath);
-            ExpandResponse response = new ExpandResponse { Nodes = new NodeInfo[] { }, ErrorMessage = node.ErrorMessage, SessionId = session.Uri, NodePath = nodePath };
+            ExpandResponse response = null;
+
+            if (node == null)
+            {
+                response = new ExpandResponse { Nodes = new NodeInfo[] { }, ErrorMessage = string.Empty, SessionId = session.Uri, NodePath = nodePath };

```

Comment:
Please add a comment explaining this scenario, basically `// This node was likely returned from a different node provider. Ignore expansion and return an empty array since we do not need to add any nodes under this section of the tree`